plugins/Lv2Instrument/Lv2Instrument.h

```diff
@@ -48,6 +48,7 @@ class Lv2InsView;
 
 }
 
+// TODO: Add support for pin connector and a variable number of audio input/output ports
 class Lv2Instrument : public Instrument, public Lv2ControlBase
```

Comment:
Is it correct that none of the Instruments (not even MIDI based ones) have the interface with the new pin connector implemented? I just look for an example, but it seems there is none yet.

Actually, for mono `Lv2Instrument`s, I may even want a pin connector. So maybe we could add this for MIDI-based instruments?

Reply:
Vestige and ZynAddSubFx both support the pin connector, though only Vestige provides a pin connector button so users can access the pin connector GUI. This is because we haven't decided on where the pin connector button should go for instruments.

None of the other instruments support the pin connector yet, though Carla, Lv2, CLAP, and OpulenZ are midi-based and could potentially add pin connector support in the future. The reason I decided to refrain from giving them pin connector support right now is because the pin connector is not really helpful for instruments unless that instrument supports multiple inputs/outputs and only VST supports that at the moment. Plus the aforementioned issue of no agreed upon location for the pin connector button for instruments.

Reply:
There aren't any instruments using the default buffer, though OpulenZ could if we wanted to give it pin connector support.